staging/src/k8s.io/api/resource/v1alpha2/types.go

```diff
@@ -19,9 +19,16 @@ package v1alpha2
 import (
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
 )
 
+const (
+	// Finalizer is the finalizer that gets set for claims
+	// which were allocated through a builtin controller.
+	Finalizer = "dra.k8s.io/delete-protection"
```

Comment:
Yes and no. kube-scheduler and kube-controller-manager need to agree on this name because one sets it and the other removes it. Users don't need to know about it and it also doesn't go into the OpenAPI, but IMHO this file is still the right place to define this constant.